src/controllers/playback/videoosd.js

```diff
@@ -1230,6 +1230,9 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
                     shell.disableFullscreen();
                 });
 
+                if (playerPauseClickTimeout) {
+                    clearTimeout(playerPauseClickTimeout);
```

Comment:
The declaration `let playerPauseClickTimeout;` should happen before it's usage here. Also it's not necessary to check for it to have a value, `clearTimeout()` will not through an exception on an invalid input. https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/clearTimeout#Notes